Library/Homebrew/software_spec.rb

```diff
@@ -391,23 +402,48 @@ def tab_attributes
 
   private
 
-  def github_packages_manifest_resource
-    return if @resource.download_strategy != CurlGitHubPackagesDownloadStrategy
+  # Resource for Docker manifests
+  def docker_manifest_resource(resource_name, basename, image_name, digest)
+    resource = Resource.new(resource_name)
+    resource.version(@version_rebuild)
+
+    resource.url("#{root_url}/#{image_name}/manifests/#{digest}", {
+      using: DockerRegistryDownloadStrategy,
+    })
+    resource.downloader.resolved_basename = basename
+    resource
+  end
+
+  # Resource for platform-specific manifest
+  def docker_bottle_manifest_resource(manifest)
+    return if @resource.download_strategy != DockerRegistryDownloadStrategy
+
+    @docker_bottle_manifest_resource ||= begin
+      @version_rebuild ||= GitHubPackages.version_rebuild(@resource.version, @rebuild)
+      @image_name ||= GitHubPackages.image_formula_name(@name)
+      digest = manifest["digest"]
+      manifest_resource = docker_manifest_resource(
+        "#{name}_bottle_manifest",
+        "#{name}-#{@version_rebuild}.bottle_manifest.json",
+        @image_name, digest
+      )
+      manifest_resource&.downloader&.mimetype = manifest["mediaType"]
+      manifest_resource&.fetch(verify_download_integrity: false)
+      JSON.parse(manifest_resource.cached_download.read) if manifest_resource&.downloaded?
+    end
+  end
```

Comment:
Why is another manifest needed when it wasn't before?

Reply:
I noted it in the PR description: 

> Fetch the platform-specific manifest is as well so that registries which use on-demand mirroring (e.g. Artifactory) can fetch the blobs from that manifest

Reply:
I see, thanks. This is something else that shouldn't be done for GitHub Packages.